packages/viz/src/lib/style/helpers/color-bins-style.ts

```diff
@@ -0,0 +1,35 @@
+import { getColors, validateParameters } from './utils';
+
+export function colorBinsStyle(
+  featureName: string,
+  { bins = defaultOptions.bins, binColors = defaultOptions.binColors }: ColorBinStyleOptions = defaultOptions
+) {
+  validateParameters(featureName, bins, binColors);
+
+  const ranges = [...bins, Number.MAX_SAFE_INTEGER];
+  const rgbaColors = getColors(binColors, bins.length);
+
+  const getFillColor = (feature: Record<string, any>) => {
+    const featureValue: number = feature.properties[featureName];
+
+    // If we want to add various comparisons (<, >, <=, <=) like in TurboCARTO
+    // we can change comparison within the arrow function to a comparison fn
+    const rangeComparison = (definedValue: number, currentIndex: number, valuesArray: number[]) =>
+      (featureValue >= definedValue) && (featureValue < valuesArray[currentIndex + 1]);
+
+    const featureValueIndex = ranges.findIndex(rangeComparison);
+    return rgbaColors[featureValueIndex];
+  };
+
+  return { getFillColor };
```

Comment:
This colorBinsStyle is, as it is, useful for fillColor, which is probably the most common request. Maybe we could make it a bit more flexible in the future (not now), so the user can choose which 'color' property he will get ('getFillColor' by default vs 'getColor')... but I would wait a bit before introducing more complexity

Reply:
Yes, I have just added the most common style specification that all (or almost all) Deck.gl layers use, so that the developer can swap layers and get what they expect.